website/docs/d/event_triggers.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: event_triggers"
+sidebar_current: "docs-mongodbatlas-datasource-event-triggers"
+description: |-
+    Describes an Event Triggers.
+---
+
+# mongodbatlas_event_triggers
+
+`mongodbatlas_event_triggers` describe all Event Triggers.
+
+## Example Usage
+
```

Comment:
any reason we don't have the example create before this like the other data source example?

Reply:
no reason, I added the example of resource before using data source